src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1233,6 +1233,29 @@ def test_aks_create_none_private_dns_zone(self, resource_group, resource_group_l
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_create_private_cluster_public_fqdn(self, resource_group, resource_group_location):
+        # reset the count so in replay mode the random names will start with 0
+        self.test_resources_count = 0
+        # kwargs for string formatting
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name
+        })
+
+        # create
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} ' \
+                     '--node-count=1 --generate-ssh-keys ' \
+                     '--enable-public-fqdn --enable-private-cluster --aks-custom-headers AKSHTTPCustomFeatures=Microsoft.ContainerService/EnablePrivateClusterPublicFQDN'
```

Comment:
Could you add some tests for the cases that `--enable-public-fqdn`, `--disable-public-fqdn` or `--public-fqdn` is False?

Reply:
add update test for `--disable-public-fqdn`